substrate/frame/core-fellowship/src/benchmarking.rs

```diff
@@ -54,11 +54,12 @@ mod benchmarks {
 	}
 
 	fn set_benchmark_params<T: Config<I>, I: 'static>() -> Result<(), BenchmarkError> {
+		let max_rank: usize = T::Members::max_rank().into();
 		let params = ParamsType {
-			active_salary: [100u32.into(); 9],
-			passive_salary: [10u32.into(); 9],
-			demotion_period: [100u32.into(); 9],
-			min_promotion_period: [100u32.into(); 9],
+			active_salary: BoundedVec::try_from(vec![100u32.into(); max_rank]).unwrap(),
+			passive_salary: BoundedVec::try_from(vec![10u32.into(); max_rank]).unwrap(),
+			demotion_period: BoundedVec::try_from(vec![100u32.into(); max_rank]).unwrap(),
```

Comment:
I think there is a `bounded_vec!`.

Reply:
Done

Reply:
CI seems unhappy with this change, I see `BoundedVec::try_from` being used in benchmarking files in the Substrate module...

Reply:
Yea its fine either way.